checker/examples/MavenExample/pom.xml

```diff
@@ -92,7 +95,10 @@
               <compilerArgs>
                   <!-- location of the annotated JDK, which comes from a Maven dependency -->
                   <arg>-Xbootclasspath/p:${annotatedJdk}</arg>
-                  <arg>-J-Xbootclasspath/p:${typeAnnotationsJavac}</arg>
+                  <!--treat Checker Framework errors as warnings-->
```

Comment:
I think the previous behavior was to not stop the build because of Checker Framework errors.  It also did not stop the build if there were any javac errors, too; so I made this change.  

I just pushed a change that comments out the line below, and I'm also checking to make sure the release sanity checks don't depend on this behavior.